### ISSUE TITLE
Incorrectly renames blocks if file paths are non-standard

### ISSUE THREAD

**Issue by lmontgomery-lyonscg** [rhoerr/supee-6788-toolbox#5]
I had this happen when one of the projects I work on had the Webshopapps_Wsalogger module installed.  It has a path Webshopapps/Wsalogger/Adminhtml/Block which appears to confuse the script in the _moveAdminControllers function.

For example it renamed Webshopapps_Wsalogger_Adminhtml_Block_Log_Grid_Renderer_Actions to Webshopapps_Wsalogger_Adminhtml_Wsalogger_Block_Log_Grid_Renderer_Actions.

The module in question is on Github as well: https://github.com/webshopapps/wsalogger

**Comment by rhoerr** [comment]
Yes, I've seen that as well. It happens because that module breaks convention on its blocks: Webshopapps/Wsalogger/Adminhtml/Block/... rather than Webshopapps/Wsalogger/Block/Adminhtml/... .

Not much to do about that short of whitelisting those block files before running.
